indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -419,6 +419,9 @@ def get_track_reviewing_state(self, track):
         else:
             return AbstractReviewingState.mixed
 
+    def get_track_reviews(self, track):
```

Comment:
Do we really need this? It doesn't add much and I don't think we want to fill the models with trivial methods.